pgmuvi/lightcurve.py

```diff
@@ -328,6 +328,41 @@ def __init__(self, xdata, ydata, yerr=None,
         self.__FITTED_MAP = False
         self.__FITTED_MCMC = False
 
+    @classmethod
+    def from_table(cls, VOTable, **kwargs):
+        """Instantiate a Lightcurve object with
+        data read in from a VOTable.
+
+        Parameters
+        ----------
+        VOTable : str
+            Name (with extension) of the VOTable containing input data
+        kwargs:
+            Arguments to be passed to the Lightcurve call
+        
+        Returns
+        ----------
+        Lightcurve object
+        """
+        from astropy.table import Table
+        data = Table.read(VOTable, format='votable')
+        c = data.colnames
+        if ('x' not in c) or ('y' not in c):
+            raise ValueError('Table must have columns x, y, and yerr')
```

Comment:
I wonder if this error message might be a bit misleading, since yerr is optional?

Reply:
Would it be better to have separate checks for each variable? And maybe some keywords that the user can use to specify which columns to use instead of forcing particular names on them?

Reply:
> I wonder if this error message might be a bit misleading, since yerr is optional?

Ooops, forgot to take `yerr` out of the error message. Was checking for it until I realised that it is optional.

> Would it be better to have separate checks for each variable? And maybe some keywords that the user can use to specify which columns to use instead of forcing particular names on them?

Yeah, good idea. I can make these changes.

Reply:
Fixed.